baking/src/tezos_baking/tezos_setup_wizard.py

```diff
@@ -74,14 +74,89 @@
 """
 
 
-def fetch_snapshot(url):
+def fetch_snapshot(url, sha256=None):
+
+    filename = os.path.join(TMP_SNAPSHOT_LOCATION, "octez_node.snapshot")
+    dirname = TMP_SNAPSHOT_LOCATION
+    metadata_file = os.path.join(dirname, "octez_node.snapshot.sha256")
+
+    # creating closures
+    def dump_metadata(metadata_file=metadata_file, sha256=sha256):
+        if sha256:
+            with open(metadata_file, "w+") as f:
+                f.write(sha256)
+        else:
+            proc_call(f"rm -f {metadata_file}")
```

Comment:
```suggestion
            os.remove(metadata_file)
```

Reply:
it throws exception when file does not exist, but we don't care if it does not exist. `proc_call`  looks much cleaner here